src/Model/AuditManagerInterface.php

```diff
@@ -21,27 +21,25 @@ interface AuditManagerInterface
     /**
      * Set AuditReaderInterface service id for array of $classes.
      *
-     * @param string $serviceId
+     * @param string[] $classes
+     *
+     * @phpstan-param class-string[] $classes
```

Comment:
Do we need `@phpstan-param` when `@param` is set?

Reply:
yes, phpstan param is more specific and compatible with phpstan